src/views/Hero/index.view.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import PostcardStack from "components/PostcardStack/index.view";
+import EmailSubscriptionForm from "components/EmailSubscription/index.view";
+import "./Hero.scss";
+
+type HeroProps = {
+  pageName: string;
+  title: string;
+  description: string;
+};
+
+const HeroView: React.FC<HeroProps> = ({ pageName, title, description }) => {
+  return (
+    <div
+      className={cx("HeroView", { HeroView__homeView: pageName === "Home" })}
+    >
```

Comment:
Using this `classnames` package, we will be able to conditionally render the backgrounds based on the theme we are on.

Reply:
Not sure if I've seen this elsewhere in the repository, but can we condense this to one line?